.golangci-strict.yml

```diff
@@ -34,37 +34,36 @@ linters:
     #- unparam # Reports unused function parameters
     - unused # (megacheck) Checks Go code for unused constants, variables, functions and types
   disable:
-    - asciicheck # Simple linter to check that your code does not contain non-ASCII identifiers
-    - depguard # Go linter that checks if package imports are in a list of acceptable packages
-    - dogsled # Checks assignments with too many blank identifiers # (e.g. x, _, _, _, := f())
-    - err113 # Golang linter to check the errors handling expressions
-    - funlen # Tool for detection of long functions
-    - gochecknoglobals # Checks that no globals are present in Go code
-    - gochecknoinits # Checks that no init functions are present in Go code
-    - gocognit # Computes and checks the cognitive complexity of functions
-    - gocyclo # Computes and checks the cyclomatic complexity of functions
-    - godot # Check if comments end in a period
-    - godox # Tool for detection of FIXME, TODO and other comment keywords
-    - gofmt # Gofmt checks whether code was gofmt-ed. By default this tool runs with -s option to check for code simplification
-    - gomodguard # Allow and block list linter for direct Go module dependencies.
-    - lll # Reports long lines
-    - nestif # Reports deeply nested if statements
-    - nolintlint # Reports ill-formed or insufficient nolint directives
-    - rowserrcheck # checks whether Err of rows is checked successfully
-    - stylecheck # Stylecheck is a replacement for golint
-    - testpackage # linter that makes you use a separate _test package
-    - whitespace # Tool for detection of leading and trailing whitespace
-    - wsl # Whitespace Linter - Forces you to use empty lines!
+    # - asciicheck # Simple linter to check that your code does not contain non-ASCII identifiers
+    # - depguard # Go linter that checks if package imports are in a list of acceptable packages
+    # - dogsled # Checks assignments with too many blank identifiers # (e.g. x, _, _, _, := f())
+    # - err113 # Golang linter to check the errors handling expressions
+    # - funlen # Tool for detection of long functions
+    # - gochecknoglobals # Checks that no globals are present in Go code
+    # - gochecknoinits # Checks that no init functions are present in Go code
+    # - gocognit # Computes and checks the cognitive complexity of functions
+    # - gocyclo # Computes and checks the cyclomatic complexity of functions
```

Comment:
There's a bunch of commits being pulled into this PR that are already in `main` - it looks like a bad rebase to me